packages/css/src/components/accordion/accordion.scss

```diff
@@ -19,9 +19,16 @@
   margin-inline: 0;
 }
 
+@mixin reset-button {
+  border: 0;
+  margin-block: 0; // [1]
+  margin-inline: 0; // [1]
+
+  // [1] Remove the margin in older Safari.
```

Comment:
Which versions of Safari render this border, and what is their browser share?

Do newer versions not require the reset at all?

Reply:
Safari 15.6 and older render this margin, which is [about 1.6% of browsers used](https://caniuse.com/usage-table). Newer versions do not have this margin. I've seen mentions that older Firefox has this margin as well, but I can't reproduce that.

Reply:
Alright, that seems enough to include these lines.